source/Weapon.h

```diff
@@ -340,6 +344,7 @@ inline const Point &Weapon::HardpointOffset() const { return hardpointOffset; }
 
 inline double Weapon::Turn() const { return turn; }
 inline double Weapon::Inaccuracy() const { return inaccuracy; }
+inline double Weapon::InaccuracySmoothness() const { return inaccuracySmoothness; }
```

Comment:
Fair assessment, I'll work on that.  Do you mean a struct or whole 'nother class with its own cpp file, though?

Reply:
Or maybe those `it.first` and `it.second` things I sometimes see.

Reply:
Addressed in 6c0cc108a2d9d53061bb9cb1665026d37fd2c2d1.

Reply:
Addressed again in ec6fc033d3524213bd899f7b41b0eaaee4b58d07.  I'm getting some feedback saying that tuples are bad because of the lack of naming (I agree, I just come up with another solution that seemed to fit your request) so I don't know if tuples were what you were suggesting.  I didn't consider a struct yet so when I have time I'll see how that look after I tie it all together but if you could clarify this then that would be helpful.